.circleci/config.yml

```diff
@@ -172,6 +174,19 @@ workflows:
       - build_web:
           requires:
             - checkout_environment
+      # Run internal branches on 4 machines
+      - test_e2e:
+          requires:
```

Comment:
you should also give this a name, like 
```
name: parallel E2E
```
and for pull requests give a different name to make it obvious. Otherwise Circle will name them `test_e2e-1` and `test_e2e-2` and it is very unclear

Reply:
Hmm that is a little bit weird though, I want them to be named the same so I can mark them as "required to pass" on GitHub 🤔 

If they are named differently I cannot do that, right?